validator/validator-main.protoascii

```diff
@@ -4624,6 +4624,17 @@ tags: {
   html_format: AMP4EMAIL
   html_format: ACTIONS
   tag_name: "TEXTAREA"
+  attrs: {
+    name: "autoexpand"
+    requires_extension: "amp-form"
+  }
+  attrs: {
+    name: "autoshrink-disabled"
```

Comment:
Is there any precedence for negative boolean attributes? If not, can we use `autoshrink` that defaults to true instead and require `autoshrink=false` to override?

Reply:
I feel like `blah=false` attribute values are ambiguous and not in line with standard DOM attributes, since you have attributes like `disabled where `<input disabled>`, `<input disabled="disabled">` and `<input disabled="false">` all disable the input, including `disabled=false`. Same with `checked` and `selected`.

I think the API will be cleaner if I remove the `autoshrink-disabled` and re-add it with a different name if there is a demand for it. The vast majority of use-cases should use autoshrink.